airflow/contrib/hooks/gcp_sql_hook.py

```diff
@@ -437,8 +442,8 @@ def _download_sql_proxy_if_needed(self):
             download_url = CLOUD_SQL_PROXY_VERSION_DOWNLOAD_URL.format(
                 self.sql_proxy_version, system, processor)
         proxy_path_tmp = self.sql_proxy_path + ".tmp"
-        self.log.info("Downloading cloud_sql_proxy from {} to {}".
-                      format(download_url, proxy_path_tmp))
+        self.log.info("Downloading cloud_sql_proxy from %s to %s" %
```

Comment:
It needs to be like the below one

```suggestion
        self.log.info("Downloading cloud_sql_proxy from %s to %s", download_url, proxy_path_tmp)
```

Reply:
Ah right indeed

Reply:
My bad. Sorry

Reply:
Stupid me - it would not indeed work this way for optimising parameter evaluation, quite obvious but somehow I missed it. It's fixed now in latest push